sdk/tools/gen_baseaddress.py

```diff
@@ -15,6 +15,9 @@
 
 Multiple directories can be specified:
 `python gen_baseaddress r:/build/msvc r:/build/gcc`
+
+Specify -64 as first argument to use 64 bit addresses:
+`python gen_baseaddress -64 C:\\Users\\Mark\\reactos\\output-MinGW-x64`
```

Comment:
I think those example paths could be "standardized"

Reply:
DOn't blame me, Copilot came up with this text 😄